.github/workflows/build_navitia_packages_for_release.yml

```diff
@@ -1,6 +1,8 @@
 name: Build Navitia Packages For Release
 
 on:
+  # for test, TO BE REMOVED BEFORE MERGE
+  pull_request:
```

Comment:
Are you sure you want to keep those lines ?

Reply:
I'll remove it, i just want to trigger these jobs on pull request